ydb/core/cms/console/console_configs_manager.cpp

```diff
@@ -998,4 +1066,12 @@ void TConfigsManager::HandleUnauthorized(TEvConsole::TEvSetYamlConfigRequest::TP
         /* success = */ false);
 }
 
+void TConfigsManager::SendInReply(const TActorId& sender, const TActorId& icSession, std::unique_ptr<IEventBase> ev) {
+    auto h = std::make_unique<IEventHandle>(sender, SelfId(), ev.release(), 0, 0);
```

Comment:
it's a good practice to preserve Cookie too (from the original message)
but it's okay to keep it zero if Console does that for other messages too